docs/product/components/editor.html

```diff
@@ -1,6 +1,7 @@
 ---
 layout: page
 title: Editor
+razor: https://razor.stackoverflow.design/components/editor-facade
```

Comment:
Technically speaking, the editor itself does not have RC support, so I'm not 100% sure if we want to mark it here.

No changes necessary, just pontificating.

Reply:
I removed it for now. We can always add the badge later on.